src/components/internal/DatePicker.tsx

```diff
@@ -8,7 +8,7 @@ import "./DatePicker.css";
 export interface DatePickerProps {
   value?: Date;
   onSelect: (value: Date) => void;
-  disabledDays?: Modifier;
+  disabledDays?: Modifier | Modifier[];
```

Comment:
**thought:** Could we make it only accept an array? We'd need to update existing implementations of this, but I think SchedulesV2's `DateCellField` is the only user of this at the moment.

Reply:
👍 Updated!